src/styles/_header.scss

```diff
@@ -37,5 +39,11 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+
+    transition-duration: $general-duration-for-transition;
```

Comment:
you should always specify transition-property, and it's easier to use shorthand transition